pr_testing/run-pr-unittests.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -ex
+cd $WORKSPACE
+$CMSSW_CVMFS_PATH/../install.sh
+export CMSSW_IB=$(basename $CMSSW_CVMFS_PATH)
+export CMSSW_VERSION=${CMSSW_IB}
+export CMSSW_CVMFS_PATH=""
+
+source $(dirname $0)/setup-pr-test-env.sh
+cd $WORKSPACE/${CMSSW_VERSION}
+CMSSW_PKG_COUNT=$(ls -d $LOCALRT/src/*/* | wc -l)
+
+rm -f ${RESULTS_DIR}/unittestGPU.txt
+mark_commit_status_all_prs 'unittests/gpu' 'pending' -u "${BUILD_URL}" -d "Running tests" || true
+echo '--------------------------------------'
+mkdir -p $WORKSPACE/gpuUnitTests
+let UT_TIMEOUT=7200+${CMSSW_PKG_COUNT}*20
+UTESTS_CMD="USER_UNIT_TESTS=cuda timeout ${UT_TIMEOUT} scram b -v -k -j ${NCPU}  unittests "
+echo "LD_LIBRARY_PATH: ${LD_LIBRARY_PATH}"
+scram build echo_LD_LIBRARY_PATH || true
+scram build -r echo_CXX || true
+cms_major=$(echo ${CMSSW_IB} | cut -d_ -f2)
+cms_minor=$(echo ${CMSSW_IB} | cut -d_ -f3)
+cms_ver="$(echo 00${cms_major} | sed -E 's|^.*(..)$|\1|')$(echo 00${cms_minor} | sed -E 's|^.*(..)$|\1|')"
+echo $UTESTS_CMD > $WORKSPACE/gpuUnitTests/log.txt
+(eval $UTESTS_CMD && echo 'ALL_OK') > $WORKSPACE/gpuUnitTests/log.txt 2>&1 || true
+echo 'END OF UNIT TESTS'
+echo '--------------------------------------'
+
+TEST_ERRORS=$(grep -ai 'had errors\|recipe for target' $WORKSPACE/gpuUnitTests/log.txt | sed "s|'||g;s|.*recipe for target *||;s|.*unittests_|---> test |;s| failed$| timeout|" || true)
+TEST_ERRORS=`grep -ai "had errors" $WORKSPACE/gpuUnitTests/log.txt` || true
+GENERAL_ERRORS=`grep -a "ALL_OK" $WORKSPACE/gpuUnitTests/log.txt` || true
+
+if [ "X$TEST_ERRORS" != "X" -o "X$GENERAL_ERRORS" = "X" ]; then
+  echo "Errors in the gpu unit tests"
+  echo 'GPU_UNIT_TEST_RESULTS;ERROR,Unit Tests,See Log,gpuUnitTests' >> ${RESULTS_DIR}/unittestGPU.txt
+  ALL_OK=false
+  UNIT_TESTS_OK=false
+  $CMS_BOT_DIR/report-pull-request-results PARSE_GPU_UNIT_TESTS_FAIL -f $WORKSPACE/gpuUnitTests/log.txt --report-file ${RESULTS_DIR}/14-unittestGPU-report.res ${REPORT_OPTS}
```

Comment:
looks like this is not work , in jenkins job I see message like `I need a report url/report file to continue` . what is `${REPORT_OPTS}`  expand to for normal unit tests?